lib/combineNodes.js

```diff
@@ -265,123 +261,12 @@ function meshHasConflict(gltf, mesh, compareMesh) {
         var primitive = primitives[i];
         for (var j = 0; j < comparePrimitivesLength; j++) {
             var comparePrimitive = comparePrimitives[j];
-            if (primitivesShareAttributeAccessor(primitive, comparePrimitive)) {
-                if (primitivesHaveOverlappingIndexAccessors(gltf, primitive, comparePrimitive)) {
+            if (PrimitiveHelpers.primitivesShareAttributeAccessor(primitive, comparePrimitive)) {
+                if (PrimitiveHelpers.primitivesHaveOverlappingIndexAccessors(gltf, primitive, comparePrimitive)) {
```

Comment:
Create a function for this in `PrimitiveHelpers` that is called here and inside `markPrimitiveConflicts`.